test/TestProjects/FirstTest-TypeSpec/FirstTest-TypeSpec.tsp

```diff
@@ -517,7 +513,6 @@ op enumParameter(@path p1: ProjectedEnum ): void;
 
 
 @doc("enum with projected name used in a model")
-@fixed
 @projectedName("csharp", "CsProjectedEnumInModel")
```

Comment:
we shall also clean up our usage for @projectedName in another PR.

Reply:
Sure, but we could still hold it until it is deprecated because we currently still support projected Name.